src/main/java/org/broadinstitute/hellbender/tools/spark/sv/utils/RDDUtils.java

```diff
@@ -0,0 +1,28 @@
+package org.broadinstitute.hellbender.tools.spark.sv.utils;
+
+import org.apache.spark.api.java.JavaRDD;
+import scala.Tuple2;
+
+import java.io.Serializable;
+
+public final class RDDUtils implements Serializable{
```

Comment:
This class and its method are not used anywhere, making it hard for me to understand the need for the method. Can this code be held off until it is necessary?